### PR TITLE
Declaration ellipsis

### LINKED ISSUE #3702: Ellipsis in class definition not accepted in C#
**Describe the bug**

I expect `class $A { ... }` to match any class.

**To Reproduce**

```
$ semgrep -l csharp -e 'class Foo { ... }'
semgrep error: invalid pattern

Error parsing C# pattern: (Failure "error parsing the pattern")
```

**What is the priority of the bug to you?**

- P2: regular bug that should get fixed

**Environment**

Semgrep 0.61.0 locally installed with brew.

### PR DESCRIPTION
**Pull request by Sjord**
Allow `class A { ... }` in C#.

Requires https://github.com/returntocorp/ocaml-tree-sitter-semgrep/pull/247

Closes #3702

### CODE REVIEW

**Comment by aryx** [comment]
still a draft or can I merge?

**Comment by Sjord** [comment]
This still needs a release of semgrep-c-sharp, and then the submodule in this PR needs to point to the released version of semgrep-c-sharp.

**Comment by Sjord** [comment]
This is ready for review and merge.

**Comment by Sjord** [comment]
@aryx Could you have another look at this?

**Review by aryx — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 - Add help text to sarif formatter output if defined in metadata field.
 - Update shortDescription in sarif formatter output if defined in metadata field.
 - Add tags as defined in metadata field in addition to the existing tags.
+- C#: support ellipsis in declarations (#3720)
```
Comment (Sjord) [inline CHANGELOG.md:22]: This is in the wrong place. I put this under 0.62.0, but it should have gone under Unreleased.

**Pull request merged**